enterprise/server/dockerservices.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Containerized service deployments
+description: Server-side components for premium plugins using Dockerfiles
+keywords: enterprise tinymcespellchecker spell check checker pro imagetools server
+---
+
+The {{site.enterpriseversion}} server-side components can be deployed on Docker orchestration applications such as [Kubernetes](https://kubernetes.io/), [Docker Swarm](https://docs.docker.com/engine/swarm/), or [OpenShift](https://www.openshift.com/). {{site.companyname}} provides packages containing pre-configured [`Dockerfiles`](https://docs.docker.com/engine/reference/builder/) for building and deploying the {{site.enterpriseversion}} server-side components using Docker.
+
+**Outline**
+
+- [Containerized services for Self-hosted Enterprise deployments](#containerizedservicesforself-hostedenterprisedeployments)
+- [Containerized services for individually licensed premium plugins](#containerizedservicesforindividuallylicensedpremiumplugins)
+
+
+## Containerized services for Self-hosted Enterprise deployments
+
+The following sections assist with containerizing the {{ site.productname }} services for users with licenses to the Self-hosted **{{site.productname}} Enterprise Bundle**.
+
+### Prerequisites
+
+* The [Docker Engine](https://docs.docker.com/engine/docker-overview/) is installed and running.
+* The user has Administrative or Root user access.
+* The user is either:
+  * Using a Linux, Unix or macOS operating system.
+  * Using Windows and has access to unix command line tools using [Git for Windows](https://gitforwindows.org/), [Cygwin](https://www.cygwin.com/), or the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+
+### Download and extract the TinyMCE Enterprise Bundle
```

Comment:
This looks like it'll need updating given the pricing refresh changes, as this refers to the older "Pro Bundle" instead of the "Essentials", "Premium", etc... bundles?

Reply:
@lnewson is the image proxy in all the bundles?

Reply:
Yes, it's in all of them.

Reply:
apparently yes.

Reply:
fixed